admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/components/listGrid-paginate.js

```diff
@@ -792,7 +792,10 @@
                     wrapperHeight = $tbody.closest('.select-group').outerHeight();
                 }
 
-                wrapperHeight -= $wrapper.next('.listgrid-table-footer:visible').outerHeight();
+                var footerOuterHeight = $wrapper.next('.listgrid-table-footer:visible').outerHeight();
+                if (typeof footerOuterHeight !== "undefined") {
+                    wrapperHeight -= wrapperHeight1;
```

Comment:
`wrapperHeight1` is not defined anywhere. I assume that it should be `footerOuterHeight`, instead?

Reply:
correct. thx for catching this